demos/browser/app/meetingV2/meetingV2.ts

```diff
@@ -287,7 +288,7 @@ export class DemoMeetingApp
 
   // feature flags
   enableWebAudio = false;
-  logLevel = LogLevel.INFO;
+  logLevel = LogLevel.DEBUG;
```

Comment:
If done with testing, change this back to INFO :) ?

Reply:
Updated :)